homeassistant/components/livisi/__init__.py

```diff
@@ -16,7 +16,9 @@
 from .const import DOMAIN, SWITCH_PLATFORM
 from .coordinator import LivisiDataUpdateCoordinator
 
-PLATFORMS: Final = [SWITCH_PLATFORM]
+# PLATFORMS: Final = [SWITCH_PLATFORM]
+
+PLATFORMS = ["switch", "binary_sensor"]
```

Comment:
Don't use magic strings, import the constants instead

Reply:
Fixed in commit 7eff9d95c97bd05e65ceed363be9e2c318553409